pkg/devspace/build/builder/buildkit/buildkit.go

```diff
@@ -204,8 +205,28 @@ func buildWithCLI(ctx context.Context, dir string, context io.Reader, writer io.
 			return fmt.Errorf("error retrieving minikube environment with 'minikube docker-env --shell none'. Try setting the option preferMinikube to false: %v", err)
 		}
 	}
+	err = command2.CommandWithEnv(ctx, dir, writer, writer, context, minikubeEnv, command[0], completeArgs...)
+	if err != nil {
+		return err
+	}
+	//load image if it is a kind-context
+	if skipPush && kubeClient != nil && kubectl.IsKindContext(kubeClient.CurrentContext()) {
+		if len(options.Tags) > 0 {
+			for _, tag := range options.Tags {
+				var stdout, stderr bytes.Buffer
+				cmd := exec.Command("kind", "load", "docker-image", tag)
+				cmd.Stdout = &stdout
```

Comment:
I think we should use writer here as inside the other command. Besides it would be good to use the `command2` package here.